index.ts

```diff
@@ -1381,7 +1382,22 @@ export class SmartContract {
 
   call(name: string, params: Array<EthereumValue>): Array<EthereumValue> {
     let call = new SmartContractCall(this._name, this._address, name, params)
-    return ethereum.call(call)
+    let result = ethereum.call(call)
+    assert(
+      result != null,
+      'Call reverted, consider using `try_' + name + '` to handle this in the mapping.',
```

Comment:
Let's make this even more useful by mentioning that this usually means an `assert`/`require` in the contract has failed. Not everyone will understand what "reverted" means in this case.